aviary/interface/test/test_height_energy_mission.py

```diff
@@ -252,6 +253,48 @@ def test_custom_phase_builder_error(self):
                 optimizer='SLSQP',
             )
 
+    def test_support_constraint_aliases(self):
+        # Test specification of multiple constraints on a single variable.
+        modified_phase_info = self.phase_info.copy()
+        modified_phase_info['climb']['user_options']['constraints'] = {
+            'throttle_1': {
+                'target': Dynamic.Mission.THROTTLE,
+                'equals': 0.2,
+                'loc': 'initial',
+                'type': 'boundary',
+            },
+            'throttle_2': {
+                'target': Dynamic.Mission.THROTTLE,
+                'equals': 0.8,
+                'loc': 'final',
+                'type': 'boundary',
+            },
+        }
+
+        prob = AviaryProblem()
+
+        csv_path = "models/test_aircraft/aircraft_for_bench_FwFm.csv"
+
+        prob.load_inputs(csv_path, modified_phase_info)
+        prob.check_and_preprocess_inputs()
+        prob.add_pre_mission_systems()
+        prob.add_phases()
+        prob.add_post_mission_systems()
+        prob.link_phases()
+
+        prob.setup()
+        prob.set_initial_guesses()
+
+        prob.run_model()
+
+        prob_vars = prob.list_problem_vars()
+        cons = {key: val for (key, val) in prob_vars['constraints']}
+        con1 = cons['traj.phases.climb->initial_boundary_constraint->throttle_1']
+        con2 = cons['traj.phases.climb->final_boundary_constraint->throttle_2']
+
+        self.assertEqual(con1['name'], 'timeseries.throttle_1')
+        self.assertEqual(con2['name'], 'timeseries.throttle_2')
```

Comment:
So this checks that both constraints were added to the problem by checking the name? Can we add an assert that the actual throttle values at start and end of phase match the constraints to make sure they are both active?

Reply:
I initially tried to do that, but I couldn't find a case that didn't end up overconstrained when I added these on the throttle. We just haven't come up with a useful solvable case where we need this.